tests/integration/API/RequestTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Tests\API;
+
+use SkyVerge\WooCommerce\Facebook\API\Request;
+
+/**
+ * Tests the API\Request class.
+ */
+class RequestTest extends \Codeception\TestCase\WPTestCase {
+
+
+	/** @var \IntegrationTester */
+	protected $tester;
+
+
+	public function _before() {
+
+		parent::_before();
+
+		require_once 'includes/API/Request.php';
+	}
+
+
+	/** Test methods **************************************************************************************************/
+
+
+	/**
+	 * @see Request::__construct()
+	 *
+	 * @param string $object_id object ID
+	 * @param string $path endpoint route
+	 * @param string $method HTTP method
```

Comment:
Missing the last param.

Reply:
Fixed in a6d0ab3. Thanks for catching those oversights.